### ISSUE TITLE
DynaTech Growth Chamber Exotic Gardenes hiány

### ISSUE THREAD

**Issue by VdszHun** [andris155/TheOld-Crafters#4982]
**Szerver**: SFSurvival

*Út a hibához:* A DynaTech addonban található Growth Chamber és Growth Chamber MK 2 képes Exotic Gardenes növények termesztésére is, viszont felfigyeltem arra, hogy nem az összes exotic gardenes növény található meg a listán, és a hiányzó növényeket nem is ismerik fel a gépek, ezt teszt szerveren is megnéztem ugyanezzel az eredménnyel.

A hiányos növények listája: 

- Tea Leaf
- Sweet Potato
- Curry Leaf
- Black Pepper
- Red Bell Pepper
- Oak Apple
- Dragon Fruit

Ha tippelnem kéne miért hiányozhatnak ezek a növények, nagyvalószínűséggel a Growth Chamberek kódolásának idejében még ezek nem létezhettek és nem kerültek be azóta se. A Slimefun Discordon nem találtam erről témát, sőt még a Slimefun Addon Community Discordon se, lehet megpróbálom megkérdezni ott, mert ott külön csatornája is van a DynaTech-nek, de számítok arra, hogy elküldenek azzal, hogy adjak több infót.

![image](https://user-images.githubusercontent.com/68443484/217047196-39295968-eb2b-4235-a1df-b61c2e2aa5e6.png)

> *Dátum és idő:*
Felfedeztem ekkor: 2023. 02. 06.

**Comment by Sanszi** [comment]
Igen, esetleg írj nekik discordra. Nem szerveroldali hibának kellene lennie szerintem, hacsak nincsenek letiltva sfben valamiért.

**Comment by VdszHun** [comment]
> Igen, esetleg írj nekik discordra. Nem szerveroldali hibának kellene lennie szerintem, hacsak nincsenek letiltva sfben valamiért.

Rendben, remélhetőleg nem fognak kérni olyan dolgokat amit nem tudok megadni. '^^

**Comment by Sanszi** [comment]
> > Igen, esetleg írj nekik discordra. Nem szerveroldali hibának kellene lennie szerintem, hacsak nincsenek letiltva sfben valamiért.
> 
> Rendben, remélhetőleg nem fognak kérni olyan dolgokat amit nem tudok megadni. '^^

Max leírod nekünk, ha van olyan amit nem tudsz. Aztán ha publikus info megadjuk szerintem

**Comment by VdszHun** [comment]
> > > Igen, esetleg írj nekik discordra. Nem szerveroldali hibának kellene lennie szerintem, hacsak nincsenek letiltva sfben valamiért.
> > 
> > Rendben, remélhetőleg nem fognak kérni olyan dolgokat amit nem tudok megadni. '^^
> 
> Max leírod nekünk, ha van olyan amit nem tudsz. Aztán ha publikus info megadjuk szerintem

A DynaTech sfsurvivalon lévő build számát (pontosat az utánna lévő git kóddal) elkérhetem mielőtt megírom a gondot nekik?
Ha mást nem is, azt biztos kérnék.
